api/serializers/teacher/dashboard.py

```diff
@@ -86,3 +90,20 @@ class Meta(CommonDashboardTeacherSerializer.Meta):
             "personal_info",
             "groups",
         )
+
+
+class DashboardTeacherTransferSerializer(PersonTransferSerializer):
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        validated_attrs = super().validate(attrs)
+        to_group = validated_attrs["to_group"]
+        from_group = validated_attrs["from_group"]
+        if self.instance is not None and self.instance in to_group.teachers.all():
+            raise ConflictError(
+                f"Teacher {self.instance.pk} is already in that group {to_group.pk}"
+            )
+
+        if self.instance is not None and self.instance not in from_group.teachers.all():
+            teacher_id = self.instance.pk
```

Comment:
Do we really need this one-time variable? Why not just put self.instance.pk into f-string below?

Reply:
nit :)

Reply:
Yes, no need anymore